SCRWebService/message_handling/message_forwarder.py

```diff
@@ -92,3 +93,17 @@ def _populate_message_template(self, template_populator, supplier_message_parame
             logger.error('003', 'Exception raised during message sending: {exception}', {'exception': e})
             raise MessageSendingError(str(e))
 
+    def _parse_response(self, templator, response: str):
+        """
+        A wrapper for the message parsing done by a templator, looks for errors and returns a simple error message
+        :param templator: 
+        :param response: 
+        :return: 
+        """
+        try:
+            return templator.parse_response(response)
+        except ET.ParseError as e:
```

Comment:
dont think we need this exception handling, can just return templator.parse_response

Reply:
Just removed this method and moved the line to the `foward_message_to_mhs` method since it's only 1 line now anyway